src/target/riscv/riscv.c

```diff
@@ -5428,6 +5445,15 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 				"hw - translate vaddr to paddr by hardware, "
 				"off - no address translation."
 	},
+	{
+		.name = "autofence",
+		.handler = riscv_set_autofence,
+		.mode = COMMAND_ANY,
+		.usage = "[on|off]",
+		.help = "When on (default), some situations, such as stepi, will automatically execute fence. "
+			"When off, user needs to take care of the cache themselves, for example, using the "
+			"riscv exec_progbuf to execute fence or cmo command."
```

Comment:
Similar updates as in the documentation should be made to the help message.

For example:

```suggestion
		.help = "When on (default), OpenOCD will automatically execute fence instructions in some situations. "
			"When off, users needs to take care of memory coherency themselves, for example by using "
			"`riscv exec_progbuf` to execute fence or CMO instructions."
```

Reply:
Thanks, addressed